Workbooks/Forcepoint Cloud Secuirty Gateway workbook.json

```diff
@@ -0,0 +1,158 @@
+{
```

Comment:
workbooksMetadata.json file should have an entry for this workbook. for update please refer this link. https://github.com/Azure/Azure-Sentinel/tree/master/Workbooks

Reply:
Need to update the Workbook metadata file WorkbooksMetadata.json to include the new workbook. Please add workbook metadata definition.

Reply:
@Dlo-Bagari , Can you please update on this.